css/properties.json

```diff
@@ -4247,7 +4272,9 @@
       "grid-auto-columns",
       "grid-auto-flow",
       "grid-column-gap",
-      "grid-row-gap"
+      "grid-row-gap",
+      "column-gap",
+      "row-gap"
```

Comment:
I'm still trying to learn this, so forgive me, but... AIUI this change is to reflect the fact that the spec now uses `row-gap` and `column-gap` instead of `grid-row-gap` and `grid-column-gap`, presumably to reflect the fact that these properties are now in the box alignment spec and are thus available for flex items and multicol. Is that right?

-> https://drafts.csswg.org/css-grid/#change-2016-grid-gap

In this case should we also remove `grid-row-gap` and `grid-column-gap` from the data?

Reply:
I actually only added this two values and did not remove anything, but you right, they should be gone. However, I assume some browsers would support both. So what should we do?

I just pushed a change marking `grid-gap`, `grid-column-gap`, `grid-row-gap` as obsolete based on https://www.w3.org/TR/2016/CR-css-grid-1-20160929/